titus-server-master/src/test/java/com/netflix/titus/master/kubernetes/pod/KubePodUtilTest.java

```diff
@@ -55,4 +65,32 @@ public void testSanitizeVolumeName() {
         String name = "Ab9bac3e:6ea1:4bc3:a803:e0070ca434c3/";
         assertThat(KubePodUtil.sanitizeVolumeName(name)).matches("ab9bac3e-6ea1-4bc3-a803-e0070ca434c3--vol");
     }
+
+    @Test
+    public void testCreatePlatformSidecarAnnotations() {
+        List<PlatformSidecar> platformSidecars = Arrays.asList(
+                PlatformSidecar.newBuilder().withName("test_sc_v1").withChannel("dev").withArguments("{\"foo\":\"fromPS\"}").build()
```

Comment:
I'm confused at this test.

Can you spilt this into two tests, one called `testCreatePlatformSidecarAnnotationsPrefersAPIOverAttributes` and `testCreatePlatformSidecarAnnotationsAcceptsAttributes` ?

I think that would make it more clear what you are trying to assert